renderer/src/Renderer.cpp

```diff
@@ -339,10 +363,22 @@ void Renderer::render(sf::RenderWindow &window, const GameState &state)
 	_shader->setMat4("projection", projection);
 	_shader->setMat4("view", view);
 
+	for (size_t i = 0; i < 20; i++)
+	{
+		for (size_t j = 0; j < 20; j++)
+		{
+			glm::mat4 model = glm::mat4(1.0f);
+			model = glm::translate(model, glm::vec3(i, 0.0f, j));
+			_shader->setMat4("model", model);
+			dirt_model->draw(*_shader);
+		}
+	}
+
```

Comment:
possibly remove the floor tile builder here, there is already one implemented that dynamically adjusts to map size.